Makefile

```diff
@@ -337,19 +326,24 @@ lint: lint-code lint-actions lint-api lint-protos
 
 lint-api: $(API_LINTER)
 	@printf $(COLOR) "Linting proto API..."
-	$(call silent_exec, $(API_LINTER) --set-exit-status $(PROTO_IMPORTS) --config=$(PROTO_ROOT)/api-linter.yaml $(PROTO_FILES))
+	@./cmd/tools/getproto/run.sh --out $(API_BINPB)
```

Comment:
I don't like how we're regenerating this and deleting it constantly. This should be tracked as a target dependency that we regenerate as needed (say, when the `api-go` version in the go.mod changes)

Reply:
Ok, that's fair. The argument for not putting it in the repo is that it's a 120K binary file that changes from time to time, and is totally (+quickly) derivable from stuff in the repo. But it probably makes sense.

Reply:
We can derive it and put it in an ignored subfolder like we do the binary dependencies we install :)

Reply:
Oh I thought you meant include it in the repo. Well I already put it in .gitignore, so we could just leave it there.